kubert/src/server.rs

```diff
@@ -20,11 +27,17 @@ pub struct ServerArgs {
     #[cfg_attr(feature = "clap", clap(long, default_value = "0.0.0.0:443"))]
     pub server_addr: SocketAddr,
 
-    /// The path to the server's TLS key
+    /// The path to the server's TLS key.
+    ///
+    /// TLS requires that `kubert` be built with one of the "rustls-tls" or
+    /// "boring-tls" feature flags enabled.
     #[cfg_attr(feature = "clap", clap(long))]
     pub server_tls_key: Option<TlsKeyPath>,
 
     /// The path to the server's TLS certificate
+    ///
+    /// TLS requires that `kubert` be built with one of the "rustls-tls" or
+    /// "boring-tls" feature flags enabled.
```

Comment:
This is user-facing in help/usage messages. I don't think it's appropriate for these details to be here.